media-tv/kodi/kodi-9999.ebuild

```diff
@@ -234,28 +234,27 @@ src_configure() {
 		$(use_enable X x11)
 }
 
-src_compile() {
-	emake V=1
```

Comment:
this is not ArchLinux, build must be verbose - https://devmanual.gentoo.org/ebuild-writing/common-mistakes/#build-log-not-verbose